### PR TITLE
make imperas iss ovpsim selectable at run-time

### PR DESCRIPTION
**Pull request by strichmo**
Remove all compile-time defines for Imperas ISS (USE_ISS)
Instead create run-time option (+USE_ISS).
Requires a PR to E40X to make tracer handoff with OVP ISS also run-time selectable.  Note that this PR points to a local fork which does this.  I will PR that update as well when we review/approve this PR.
Note that due to design of step-and-compare testcases will not repeat between running with and without ISS.

@MikeOpenHWGroup Will discuss this tomorrow morning.

### CODE REVIEW

File: bin/cv_regress
Comment (MikeOpenHWGroup) [inline bin/cv_regress:83]: Is this a fix for the `YAMLLoadWarning: calling yaml.load() without Loader` warning?

**Review by MikeOpenHWGroup — APPROVED** [review]
This is good @strichmo, and delivers a capability we have long wished for.   I've added a couple of comments, but none of them should gate my approval.  I'm running a CI regression locally and will merge this in pending that and our meeting in an hour.

File: bin/templates/metrics.json.j2
Comment (strichmo) [reply bin/templates/metrics.json.j2:27]: Correct.  The containerization Metrics provides hid the fact that all test runs were recompiling the RTL.  For the most part this was harmless besides burning a few cycles, but should definitely be cleaned up as in this diff.

File: bin/templates/regress_macros.j2
Comment (strichmo) [reply bin/templates/regress_macros.j2:2]: This simply supports setting 1 or 0 in lieu of yes or no from the cv_regress command line.  Since our contributors seem to be evenly split amongst preferring YES or 1 (and I prefer 0 or 1) we support both.

File: cv32e40x/tb/uvmt/uvmt_cv32e40x_tb_ifs.sv
Comment (strichmo) [reply cv32e40x/tb/uvmt/uvmt_cv32e40x_tb_ifs.sv:180]: The entirety of this code block is setting an override file purely for the OVPSim model.  I believe a self-checking test would need to separately parse the plusargs to determine the bootstreap mtvec address.

File: bin/cv_regress
Comment (strichmo) [reply bin/cv_regress:83]: Yes, unfortunately some older PyYAMLs cannot handle supplying the Loader argument to .load, so it must be broken up as such.  I'll add a comment as otherwise it does look bizarre.

Comment (strichmo) [reply bin/cv_regress:83]: Otherwise the newer PyYAMLs spit out a very ugly warning that litters our logs.

**Review by MikeOpenHWGroup — APPROVED** [review]
Thanks for the replies @strichmo.  It all looks good to me now.  I will approve this PR, but not merge it until you remove the DONOTMERGE label.

**Comment by strichmo** [comment]
Thanks @MikeOpenHWGroup I have a PR to the E40X design team to remove their ifdef ISS in the tracer.  Once they merge that (hopefully overnight) I will change the Common.mk in this PR, run a final ci_check and then remove the DONOTMERGE label.

**Comment by strichmo** [comment]
@MikeOpenHWGroup  These latest changes fix the yaml.load issue across all of our scripts and adds appropriate comments.

I also added an --iss option to ci_check to support:
% ci_check -s xrun --iss=0

**Review by MikeOpenHWGroup — APPROVED** [review]

**Pull request merged**
